Adding explanatory text to _infographics/climate/concentration-warming-relationship.md

willemfrishert opened the pull request:
Task of #14: Add English texts to the published infographics

### Preview Build

Current preview build is available at https://en-facts-on-climate--preview-concentration-warming-rel-s74b54da.web.app/infographics/concentration-warming-relationship

### What does this PR do?

- bulk-translated the Czech text to English using [DeepL](https://www.deepl.com) for concentration-warming-relationship.md
- Use [Grammarly](https://www.grammarly.com/) to proofread the texts.
- Introduce Glossary for English to support reference to `NOAA`.

### Commented out links:
- Commented out preview-box (using Jekyll comments) as these are (not yet) available in English
  -  [slug="otepleni-zvysenim-koncentrace-co2](https://faktaoklimatu.cz/explainery/otepleni-zvysenim-koncentrace-co2)

- Commented out link in text (using html comments) as these are (not yet) available in English:
  - [explainery/otepleni-zvysenim-koncentrace-co2](https://faktaoklimatu.cz/explainery/otepleni-zvysenim-koncentrace-co2). (same as preview-box)
  - [Evolution of the global temperature anomaly - https://faktaoklimatu.cz/infografiky/vyvoj-teplotni-anomalie](https://faktaoklimatu.cz/infografiky/vyvoj-teplotni-anomalie)
  -  [Evolution of atmospheric CO₂ concentration - https://faktaoklimatu.cz/infografiky/koncentrace-co2](https://faktaoklimatu.cz/infografiky/koncentrace-co2)

willemfrishert committed:
Added _data/glossary.yml and collections/_pages/glossary.html to support glossary links in English pages.

willemfrishert committed:
Translation and initial pass of text styling.

tjarnikova commented:
Response to Willem’s request for 2nd pair of eyes (only small edits): 

Looking at it now, I haven’t found any cases in which a switch to active voice would be clearly better (in my subjective opinion). However, there are a few places where the AI doesn’t catch minor-to-moderate errors that impede readability:
____
original:
The theoretically derived relationship for the dependence of global warming on greenhouse gas concentration is logarithmic. [1](https://en-facts-on-climate--preview-concentration-warming-rel-s74b54da.web.app/infographics/concentration-warming-relationship#fn:1) [2](https://en-facts-on-climate--preview-concentration-warming-rel-s74b54da.web.app/infographics/concentration-warming-relationship#fn:11) However, a small range of concentrations can be well approximated by a linear relationship, as shown by the measured data

corrected:
The theoretically derived relationship for the dependence of global warming on greenhouse gas concentration is logarithmic. [1](https://en-facts-on-climate--preview-concentration-warming-rel-s74b54da.web.app/infographics/concentration-warming-relationship#fn:1) [2](https://en-facts-on-climate--preview-concentration-warming-rel-s74b54da.web.app/infographics/concentration-warming-relationship#fn:11)  However, for a small range of concentrations, we can approximate this relationship reasonably well with a linear relationship, as shown by the measured data. 
Czech:
Teoreticky odvozený vztah pro závislost oteplení na koncentraci skleníkového plynu je logaritmický.[1](https://faktaoklimatu.cz/infografiky/souvislost-koncentrace-oteplovani#fn:1) [2](https://faktaoklimatu.cz/infografiky/souvislost-koncentrace-oteplovani#fn:11) V malém rozmezí koncentrací jej ale můžeme dobře aproximovat vztahem lineárním, což ukazují i naměřená data.
____

original:
Global warming shown includes all these phenomena, but only CO2 concentrations are plotted on the horizontal axis. 

corrected:
The global warming shown includes all these phenomena, but only CO2 concentrations are plotted on the horizontal axis. 
______
original
WHERE THE DATA IN THIS INFOGRAPHIC COMES FROM?
corrected
WHERE DOES THE DATA IN THIS INFOGRAPHIC COME FROM?
_____
original
The link between global warming and atmospheric carbon dioxide concentration is one of the keys and longest-studied links in the study of climate change.

corrected:
The link between global warming and atmospheric carbon dioxide concentration is one of the most crucial and longest-studied relationships in the study of climate change.
Czech:
Souvislost globálního oteplování a koncentrace atmosférického oxidu uhličitého je jednou z klíčových a nejdéle zkoumaných souvislostí v rámci studia klimatické změny.
(Note: changed sentence to have two adjectives as in original; the implied “key in the study of climate change” in the first translation is awkward)

willemfrishert committed:
Text correction.

original:
"However, a small range of concentrations can be well approximated by a linear relationship, as shown by the measured data"

corrected:
"However, for a small range of concentrations, we can approximate this relationship reasonably well with a linear relationship, as shown by the measured data."

willemfrishert committed:
Text correction.

original:
Global warming shown includes all these phenomena, but only CO2 concentrations are plotted on the horizontal axis.

corrected:
The global warming shown includes all these phenomena, but only CO2 concentrations are plotted on the horizontal axis.

willemfrishert committed:
Text correction.

original:
Where the data in this infographic comes from?

correction:
Where does the data in this infographic come from?

willemfrishert committed:
Text correction.

original
The link between global warming and atmospheric carbon dioxide concentration is one of the keys and longest-studied links in the study of climate change.

corrected:
The link between global warming and atmospheric carbon dioxide concentration is one of the most crucial and longest-studied relationships in the study of climate change.

willemfrishert commented:
Thanks for the feedback @tjarnikova. I updated this PR with the corrections are you mention above (one commit per correction).

mukrop commented:
The updated preview of this version can be found at
https://en-facts-on-climate--preview-concentration-warming-rel-s74b54da.web.app/infographics/concentration-warming-relationship
(More details on how to get this link I'll write in Slack for @willemfrishert.)

mukrop commented on collections/_pages/glossary.html:
```diff
@@ -0,0 +1,25 @@
+layout:      single-section-page
+title:       "Glossary of terms"
+slug:        glossary
+intro:       "A brief overview of the most commonly used passwords and abbreviations with a short explanation or commentary."
```
Since the glossary currently has only a single definition, can you please add a line in the intro paragraph describing that it's work in progress and more terms will be added over time?

willemfrishert replied:
I've updated the paragraph with that suggestion.

willemfrishert committed:
Update glossary intro paragraph.

tjarnikova reviewed (commented):
All changes look good.

tjarnikova reviewed (approved):
Good to go!

mukrop committed:
Delete old Czech code comment

mukrop reviewed (approved):
I've just deleted an old Czech code comment. Good job, merging and deploying.
